src/Generator.php

```diff
@@ -239,4 +239,9 @@ public function generateTemplate(string $targetPath, string $templateName, array
             $variables
         );
     }
+
+    public static function getParentClassName(): string
```

Comment:
```suggestion
    public static function getControllerBaseClass(): string
```

And really, this should return the FQCN. If we need to tweak `generateController()` to only grab the "last" piece, then let's do that.

Reply:
this I've come up with a more elegant solution than depending on a string.

Reply:
done